soda-core/tests/helpers/a_time_machine.py

```diff
@@ -0,0 +1,28 @@
+import datetime
+from typing import Optional
+
+import pytest
+
+
+class TimeMachineDatetime(datetime.datetime):
+    NOW: Optional[datetime.datetime] = None
+
+    @classmethod
+    def now(cls, tz=None) -> datetime.datetime:
+        return cls.NOW if cls.NOW else super().now(tz)
+
+
+datetime.datetime = TimeMachineDatetime
```

Comment:
please move the comment from the PR into the code here. It will get forgotten in PR, but will be super handy if/when we need to change this code

Reply:
done 👍